vocab/credentials/v2/template.html

```diff
@@ -264,33 +264,36 @@ <h2>Diagram description</h2>
           On the left side of this column, there are four labeled ellipses, styled as Class. 
           These are, from top to bottom, "VerifiableCredential", "JsonSchemaCredential",
           "VerifiableCredentialGraph", and "VerifiablePresentation". 
-          There is also a single circle, not styled as Property or Class, labeled as "OR", and another box 
-          styled as a Property labeled as "verifiableCredential". 
+          There is also a small, unlabelled circle, as a crossing point for connector lines.
         </p>
         <p>
           The "VerifiableCredential" ellipse is connected through a connecting line
-          styled as Domain to the "credentialSchema", "credentialStatus", "credentialSubject",
+          styled as Domain Of to the "credentialSchema", "credentialStatus", "credentialSubject",
           "issuer", "evidence", "refreshService", "renderMethod", and "confidenceMethod" boxes. 
```

Comment:
Note that this includes what was line 270, which was deleted above, to work around GitHub change request limitations.
```suggestion
          The "VerifiableCredential" ellipse is connected to the "credentialSchema", "credentialStatus",
          "credentialSubject", "issuer", "evidence", "refreshService", "renderMethod", and "confidenceMethod"
          boxes, through connecting lines styled as Domain Of. 
```